python-avd/pyavd/_eos_designs/schema/schema_fragments/use_cv_topology.schema.yml

```
@@ -13,3 +13,4 @@ keys:
     description: |-
       Generate AVD configurations directly from a given CloudVision topology.
       See `cv_topology` for details.
+      Requires both `cv_topology` and `cv_topology_hierarchy` to be set.
```

`cv_topology_levels`?

```suggestion
      Requires both `cv_topology` and `cv_topology_levels` to be set.
```